kie-server-parent/kie-server-services/kie-server-services-jbpm-ui/src/main/resources/form-templates-providers/js/kieserver-ui.js

```diff
@@ -340,15 +340,18 @@ function encodeImageFileAsURL(input) {
 		
 		var fileReader = new FileReader();
 
-		fileReader.onload = function(fileLoadedEvent) {
-			var local = fileLoadedEvent.target.result; // <--- data: base64
-			var srcData = local.replace(/^data:.*\/.*;base64,/, "");
-
-			fileData.get(input.id).data = srcData;
-			
-			console.log("Converted Base64 version is " + srcData);
+		fileReader.onload = function (fileLoadedEvent) {
+			if (fileReader.readyState == 2) {
+				var local = fileLoadedEvent.target.result; // <--- data: base64
+				var srcData = local.replace(/^data:.*\/.*;base64,/, "");
+
+				fileData.get(input.id).data = srcData;
+				fileReader.readAsDataURL(fileToLoad);
+				console.log("Converted Base64 version is " + srcData);
```

Comment:
I'd remove that console log to make this efficient when dealing with big docs, a part from this, the change works fine

Reply:
removed